### PR TITLE
add hkdf as dependency and correct PyPy2

### PR DESCRIPTION
**Pull request by TiberiumPY**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # Pure-Python SPAKE2
 
 * License: MIT
-* Dependencies: none (pure-python)
-* Compatible With: Python 2.6, 2.7, 3.3, 3.4, 3.5, pypy2
+* Dependencies: hkdf (pure-python)
```
Comment (asymmetric) [inline README.md]: I would loose the `(pure-python)` here.

**Comment by asymmetric** [comment]
Fixes #7.

**Comment by TiberiumPY** [comment]
@asymmetric updated

**Comment by warner** [comment]
I'm good with the first two patches.. I'll land that now. I think I'd like to leave the "Pure-Python SPAKE2" headline in the README, since not needing a C compiler is what's special about this particular library (and the HKDF dependency doesn't need a C compiler either).

thanks!

**Comment by warner** [comment]
merged first two patches in eb11820 , omitting final patch

**Pull request closed without merging**